CODEOWNERS

```diff
@@ -15,16 +15,16 @@
 /src/api/                                   @box/preview @box/ui-elements
 /src/api/box-edit/                          @box/partners
 /src/api/Feed.js                            @box/preview
-/src/api/tasks/                             @box/collab
-/src/api/uploads/                           @box/create-communicate
+/src/api/tasks/                             @box/preview
+/src/api/uploads/                           @box/preview
```

Comment:
For the `tasks` and `uploads`, I believe the correct team is Create & Communicate (the public Notes Github team)